### ISSUE TITLE
Implement "Remove as subreddit" feature for Remove Action

### ISSUE THREAD

**Issue by FoxxMD** [FoxxMD/context-mod#124]
Toolbox has done the hard work of finding this undocumented api endpoint :) It allows removing a submission/comment with "remove as subreddit" which then posts as `subreddit-modteam` user. 

https://github.com/toolbox-team/reddit-moderator-toolbox/releases/tag/v6.0.0

* https://github.com/toolbox-team/reddit-moderator-toolbox/issues/582
* https://github.com/toolbox-team/reddit-moderator-toolbox/pull/583

**Comment by FoxxMD** [comment]
```json
{
  "item_id": [
    "t1_ikodx0k"
  ],
  "message": "comment goes here",
  "title": "rule title goes here?",
  "type": "public_as_subreddit", // or `public`, `private`, `private_exposed`
  "lock_comment": false
}
```
for comments => `/api/v1/modactions/removal_comment_message` [PRAW](https://praw.readthedocs.io/en/stable/code_overview/other/commentmoderation.html#praw.models.reddit.comment.CommentModeration.send_removal_message)
for submissions => `/api/v1/modactions/removal_link_message`  [PRAW](https://praw.readthedocs.io/en/stable/code_overview/other/submissionmoderation.html#praw.models.reddit.submission.SubmissionModeration.send_removal_message)

item has to be removed prior to calling api or else reddit returns an error

**Comment by FoxxMD** [comment]
Implemented in   59d31bde847acb9fe5d47576928be57a6447130f

**Comment by FoxxMD** [comment]
Released in [0.13.3](https://github.com/FoxxMD/context-mod/releases/tag/0.13.3)
